app/experimenter/experiments/api/v5/urls.py

```diff
@@ -2,10 +2,17 @@
 from django.views.decorators.csrf import csrf_exempt
 from graphene_file_upload.django import FileUploadGraphQLView
 
+from experimenter.experiments.api.v5.views import NimbusExperimentCSVListView
+
 urlpatterns = [
     url(
         r"graphql",
         csrf_exempt(FileUploadGraphQLView.as_view(graphiql=True)),
         name="nimbus-api-graphql",
     ),
+    url(
+        r"^csv/$",
+        NimbusExperimentCSVListView.as_view(),
+        name="nimbus-experiments-api-csv",
```

Comment:
🤔 what do you think? --
```suggestion
        name="nimbus-experiments-report",
```

Reply:
idk, or maybe we need `-api-` in the name? not sure about naming conventions here

Reply:
so to keep consistent with csv. what about `nimbus-experiments-csv`?

Reply:
let's just leave it as-is for now. We can always change it later 😄